db/migrate/20211119153143_create_users.rb

```diff
@@ -0,0 +1,9 @@
+class CreateUsers < ActiveRecord::Migration[6.1]
+  def change
+    create_table :users do |t|
+      t.string :name
```

Comment:
It would be nice to add `null: false` for the username to the database migrations also